### PR TITLE
Further instructions to add apache config for PHP 7.0.

### PR DESCRIPTION
**Pull request by philcook**
@ablyler as mentioned in this conversation https://github.com/Homebrew/homebrew-php/issues/1829

There are further steps to add the apache config for PHP 7.0 so I have added these instructions to the CLI instructions.

Cheers

### CODE REVIEW

**Comment by ibnbd** [comment]
Not working ....

**Comment by ablyler** [comment]
@ibnbd please create a new issue with more details

**Comment by philcook** [comment]
@ibnbd yes just let us know whats not working...

**Comment by ibnbd** [comment]
Sorry for the late replay 
_____________________________________________________________________
bash-3.2# php -v
PHP 7.0.0RC6 (cli) (built: Oct 29 2015 13:46:05) ( NTS )
Copyright (c) 1997-2015 The PHP Group
Zend Engine v3.0.0-dev, Copyright (c) 1998-2015 Zend Technologies
with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2015, by Zend Technologies

_____________________________________________________________________

bash-3.2# brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: You have unlinked kegs in your Cellar
Leaving kegs unlinked can lead to build-trouble and cause brews that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
    php56
______________________________________________________________________

LoadModule php7_module /usr/local/opt/php70/libexec/apache2/libphp7.so

______________________________________________________________________

With php56 my localhost works perfectly, when I am using php70 my localhost always offline

**Comment by philcook** [comment]
@ibnbd looks like your CLI is working fine, the brew doctor report is nothing to worry about as you need to unlink one version of php for the other to get the CLI version working.

Now if your using local OSX apache you may need just a couple of bits of extra config in your /etc/apache2/httpd.conf

These is not require in php 5.x but are required in 7.x

Where you have
```
<IfModule dir_module>
    DirectoryIndex index.html
</IfModule>
```

Replace with
```
<IfModule dir_module>
    DirectoryIndex index.php index.html
</IfModule>
```

And finally underneath
```
<FilesMatch "^\.([Hh][Tt]|[Dd][Ss]_[Ss])">
    Require all denied
</FilesMatch>
```

Add the following
```
<FilesMatch \.php$>
    SetHandler application/x-httpd-php
</FilesMatch>
```

Basically you are explicitly telling apache to use index.php file ahead of index.html file as a first loaded file and secondly where its a file ending in .php set it to use PHP interpreter.

Hopefully that should fix your issue. If it does your the second person to contact me about this and therefore I should probably add these as checks when switching to PHP 70.

**Comment by ibnbd** [comment]
# 
# DirectoryIndex: sets the file that Apache will serve if a directory
# is requested.
#
<IfModule dir_module>
    DirectoryIndex index.php index.html
</IfModule>

# 
# The following lines prevent .htaccess and .htpasswd files from being
# viewed by Web clients.
# 
<FilesMatch "^\.([Hh][Tt]|[Dd][Ss]_[Ss])">
    Require all denied
</FilesMatch>


<FilesMatch \.php$>
    SetHandler application/x-httpd-php
</FilesMatch>

_____________________________________________________________

Did as you mentioned, still not working.

**Comment by ibnbd** [comment]
Fixed, problem was my apache config file. Thank you very much (y)

**Comment by erdegrey** [comment]
ibnbd, I am having the same problem and am new to this - how did you fix it? 
thank you!

**Comment by ibnbd** [comment]
@erdegrey hey there,

First install php using brew then modify your apache config file.
Apache config file should be at `/etc/apache2/httpd.conf`

Open your terminal app and type `sudo nano /etc/apache2/httpd.conf`
If you scroll down using your Keyboard down arrow key you will see lot's of `LoadModule`

Add this php module there

`LoadModule php7_module /usr/local/opt/php70/libexec/apache2/libphp7.so`

In case you are using php 7.1, use php 7.1 module location.

Then using scroll down and search for 

```
<IfModule dir_module>
    DirectoryIndex index.html
</IfModule>
```

Modify this with 

```
<IfModule dir_module>
    DirectoryIndex index.php index.html
</IfModule>
```


In addition you can add this 

```
<FilesMatch "^\.([Hh][Tt]|[Dd][Ss]_[Ss])">
    Require all denied
</FilesMatch>

<FilesMatch \.php$>
    SetHandler application/x-httpd-php
</FilesMatch>
```

Once you are done using your keyboard press `control + x` it will ask to save changes press `y` and enter to save your settings.

Once done you can restart apache and check your settings, by default apache document root `/Library/WebServer/Documents`. You can change this by searching documentroot value on the apache config file you opened before. You may face permission problem, make sure to right click on your new location folder and go to Get Info -> General -> check "Shared folder".

Hopefully it will be helpful for you.

**Comment by papakay** [comment]
Hello, please i'm having the same issue and i've not been able to resolve it.

I have tried all the suggestions provide and none seems to work for me.

Please see below my php version

PHP 7.0.14 (cli) (built: Dec  9 2016 07:34:25) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies

I used brew to install following brew install php70

Thank you in advance for your help.

**Comment by javian** [comment]
@papakay please open a new issue if you are having issues.
